docs/howto/object-storage/s3/object-lock.md

```diff
@@ -0,0 +1,271 @@
+# Object lock
+
+Object lock is a feature in S3 that enables you to lock your objects
+to prevent them from being deleted or overwritten. This feature provides
+a storage solution based on the **Write Once, Read Many (WORM)** model,
+which can be helpful when you want to maintain data integrity,
+comply with regulations, or retain data for a specific period.
+
+> Are you looking to set the expiration/retention time for your
+> objects? Then you should check our [Object expiry](expiry.md) guide.
+
+## Object lock modes
+
+Object lock offers two methods for managing object retention:
+**Retention periods** and **Legal hold**.
+
+### Retention periods
+
+Retention periods specify the length of time that an object
```

Comment:
```suggestion
Retention periods specify the length of time an object
```

Reply:
I prefer to leave "that" in, in this instance.